torch/_inductor/fx_passes/post_grad.py

```diff
@@ -69,6 +69,19 @@
 ]
 
 
+def apply_pass(pass_fn: Callable[[], object], name: Optional[str] = None) -> None:
+    # TODO - we should just make this part of GraphTransformObserver
```

Comment:
Yeah, we should think about how to make sure people don't forget to call this when they are doing passes